extension/webpack.config.js

```diff
@@ -37,5 +38,13 @@ module.exports = {
                 }
             ],
         }),
-    ]
+    ],
```

Comment:
As per #58 , we are moving off css loaders

Reply:
done